rpi3.mk

```diff
@@ -11,7 +11,8 @@ override COMPILE_S_KERNEL  := 64
 # Need to set this before including common.mk
 BUILDROOT_GETTY_PORT ?= ttyS0
 BR2_ROOTFS_POST_BUILD_SCRIPT ?= "board/raspberrypi3-64/post-build.sh"
-
+#For Raspberry Pi 3 B plus set to rpi3-bplus
+PLATFORM ?= rpi3-b
```

Comment:
I think I saw a comment from you somewhere that you couldn't use the composite form? Or did I mix that up? What I'm looking for here is something like
```make
PLATFORM=rpi-3b
PLATFORM=rpi-3bplus
```

Reply:
Platform directory in `optee_os` is `core/arch/arm/plat-rpi3` so PLATFORM has to be `rpi3` and `PLATFORM_FLAVOR` may be whatever we want (for instance `b` or `bplus`).
The consequence is, the "composite" form of `PLATFORM` is written `rpi3-something`.

https://github.com/OP-TEE/optee_os/blob/master/Makefile#L32-L33

Another option would be to rename `core/arch/arm/plat-rpi3` to `core/arch/arm/plat-rpi`, but since all variants are still 3 more or less, would it be justified?

Reply:
> Another option would be to rename `core/arch/arm/plat-rpi3` to `core/arch/arm/plat-rpi`, but since all variants are still 3 more or less, would it be justified?

I was thinking about the same. My initial thought was to leave the `rpi3-...` and start using `rpi-...` instead. But as long as it's only rpi3 devices we have, it might be more work than we gain by doing so. In that case, use what I think I proposed in the other github issue about this:
```make
PLATFORM=rpi3-b
PLATFORM=rpi3-bplus
```